File: Umbrella/src/main/java/com/nerdery/umbrella/data/model/HourlyResponse.java
package com.nerdery.umbrella.data.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class HourlyResponse {
    @SerializedName("data")
    List<ForecastCondition> hours;

    public List<ForecastCondition> getHours() {
        return hours;
    }
}

File: Umbrella/src/main/java/com/nerdery/umbrella/data/model/HourlyResponse.kt
package com.nerdery.umbrella.data.model

import com.google.gson.annotations.SerializedName

data class HourlyResponse(
    /**
     * Ordered List of Hourly [ForecastCondition]
     * @return List of [ForecastCondition]
     */
    @SerializedName("data")
    var hours: List<ForecastCondition>
    )
